src/stirling/source_connectors/socket_tracer/protocols/cql/types.h

```diff
@@ -102,6 +103,7 @@ struct Frame : public FrameBase {
   FrameHeader hdr;
   std::string msg;
   bool consumed = false;
+  bool discarded = false;
```

Comment:
Do we end up using this member (or did it become not-needed)?

Reply:
Good catch, this is now unneeded. I've just removed it.